tests/current_thread.rs

```diff
@@ -293,6 +294,48 @@ fn spawn_and_turn() {
     assert_eq!(2, cnt.get());
 }
 
+#[test]
+fn spawn_in_drop() {
+    let mut current_thread = CurrentThread::new();
```

Comment:
Shouldn't we create variables as closer to usage as possible? I mean `current_thread` may be created right before `current_thread.spawn(`

Reply:
Maybe? I don't have a huge opinion about it. What do the rust guidelines say?

Reply:
There is no such a rule in rust guidelines. I remember that from `Code Complete` by McConnell.

Reply:
If you want, you can submit a PR to update it.

Reply:
https://github.com/tokio-rs/tokio/pull/161